### ISSUE TITLE
Bug: Error When Loading Settings

### ISSUE THREAD

**Issue by Landelor** [SFX-WoW/Masque#80]
Hi,

I'm getting the below error when trying to load Masque settings. None of my skins are showing.

`Message: Interface\AddOns\Masque\Options\Skins.lua:457: attempt to index field 'Skins' (a nil value)
Time: Mon Oct 28 10:18:31 2019
Count: 1
Stack: Interface\AddOns\Masque\Options\Skins.lua:457: attempt to index field 'Skins' (a nil value)
Interface\AddOns\Masque\Options\Skins.lua:457: in function `UpdateSkinOptions'
Interface\AddOns\Masque\Core\Group.lua:417: in function `__Update'
Interface\AddOns\Masque\Core\Groups.lua:84: in function <Interface\AddOns\Masque\Core\Groups.lua:58>
Interface\AddOns\Masque\Core\Groups.lua:91: in function `GetGroup'
Interface\AddOns\Masque\Options\Skins.lua:425: in function `func'
Interface\AddOns\Masque\Options\Core.lua:34: in function <Interface\AddOns\Masque\Options\Core.lua:31>
Interface\AddOns\Masque\Options\Core.lua:118: in function `func'
Interface\AddOns\Masque\Options\Core.lua:34: in function <Interface\AddOns\Masque\Options\Core.lua:31>
Interface\AddOns\Masque\Options\Core.lua:132: in function `ToggleOptions'
Interface\AddOns\Masque\Masque.lua:115: in function `?'
Interface\FrameXML\ChatFrame.lua:4839: in function `ChatEdit_ParseText'
Interface\FrameXML\ChatFrame.lua:4502: in function `ChatEdit_SendText'
Interface\FrameXML\ChatFrame.lua:4538: in function `ChatEdit_OnEnterPressed'
[string "*:OnEnterPressed"]:1: in function <[string "*:OnEnterPressed"]:1>

Locals: self = <table> {
 SkinShadow = <function> defined @Interface\AddOns\Masque\Core\Regions\Shadow.lua:90
 GetID = <function> defined @Interface\AddOns\Masque\Core\Groups.lua:40
 SkinTexture = <function> defined @Interface\AddOns\Masque\Core\Regions\Texture.lua:39
 OptionsLoaded = true
 Callback = <table> {
 }
 WOW_RETAIL = true
 Locale = <table> {
 }
 SkinIcon = <function> defined @Interface\AddOns\Masque\Core\Regions\Icon.lua:33
 GetColor = <function> defined @Interface\AddOns\Masque\Core\Utility.lua:74
 Skins = <table> {
 }
 AddSkin = <function> defined @Interface\AddOns\Masque\Skins\Skins.lua:40
 API_VERSION = 80200
 SkinNewItem = <function> defined @Interface\AddOns\Masque\Core\Regions\NewItem.lua:73
 GetSize = <function> defined @Interface\AddOns\Masque\Core\Utility.lua:26
 ToggleOptions = <function> defined @Interface\AddOns\Masque\Options\Core.lua:131
 GetRegion = <function> defined @Interface\AddOns\Masque\Core\Core.lua:87
 API = <table> {
 }
 Group_MT = <table> {
 }
 GetOptions = <function> defined @Interface\AddOns\Masque\Options\Skins.lua:159
 Setup = <table> {
 }
 GetType = <function> defined @Interface\AddOns\Masque\Core\Core.lua:37
 LDBI = <table> {
 }
 UpdateSkinOptions = <function> defined @Interface\AddOns\Masque\Options\Skins.lua:453
 RegTypes = <table> {
 }
 SkinNormal = <function> defined @Interface\AddOns\Masque\Core\Regions\Normal.lua:127
 Authors = <table> {
 }
 LDBO = <table> {
 }
 OptionsPanel = <unnamed> {
 }
 SkinGloss = <function> defined @Interface\AddOns\Masque\Core\Regions\Gloss.lua:90
 GetStandAlone = <function> defined @Interface\AddOns\Masque\Options\Core.lua:165
 GetScale = <function> defined @Interface\AddOns\Masque\Core\Utility.lua:100
 Update = <function> defined @Interface\AddOns\Masque\Masque.lua:163
 SkinIconBorder = <function> defined @Interface\AddOns\Masque\Core\Regions\IconBorder.lua:73
 SkinList = <table> {
 }
 GetArg = <function> defined @Interface\AddOns\Masque\Options\Core.lua:170
 OLD_VERSION = 70200
 SkinButton = <function> defined @Interface\AddOns\Masque\Core\Button.lua:61
 db = <table> {
 }
 CleanDB = <function> defined @Interface\AddOns\Masque\Core\Groups.lua:103
 GetGroup = <function> defined @Interface\AddOns\Masque\Core\Groups.lua:89
 Groups = <table> {
 }
 Websites = <table> {
 }
 SkinMask = <function> defined @Interface\AddOns\Masque\Core\Regions\Mask.lua:27
 Options = <table> {
 }
 SkinCooldown = <function> defined @Interface\AddOns\Masque\Core\Regions\Frame.lua:105
 SkinFrame = <function> defined @Interface\AddOns\Masque\Core\Regions\Frame.lua:57
 SkinText = <function> defined @Interface\AddOns\Masque\Core\Regions\Text.lua:47
 GetTexCoords = <function> defined @Interfac`

**Comment by StormFX** [comment]
Reinstall it with the game shut down.

**Comment by Landelor** [comment]
Installed it the first time with the game shut down.
A restart also didn't sort it.

**Comment by StormFX** [comment]
The error is the result of an installation issue.

- Make sure you're using the latest version.
- Make sure you're _not_ using a no-libs version.
- With the game shut down, manually remove the _Masque_ folder from your add-ons folder and then reinstall it.

**Comment by Landelor** [comment]
Tried re-adding the addon after removing all traces of it.
Left it as the only addon running and still getting the error.

**Comment by StormFX** [comment]
I'm unable to reproduce the error. I need some more information before attempting to troubleshoot it further.

- Which version are you trying to install (ie, version number + classic/retail)
- Where are you getting the installation package (website and add-on manager, if using one).
- What exact steps are you taking that cause the error to occur.

**Comment by Landelor** [comment]
Version 8.2.7 for retail.
I tried installing through Twitch app as well as downloading from Curseforge manually.
With a freshly loaded UI I try to open the Masque options. Doesn't matter how, through the button under interface settings or using /msq.

**Comment by StormFX** [comment]
Honestly, I've no clue. This is the only report of this I've had and it's working fine on my end.

**Comment by Landelor** [comment]
Yeah. No matter what I do it doesn't work.
What would cause that object to return a nil value?
Also it's saying that it's outdated, but I assumes that's just a TOC bump.

**Comment by Landelor** [comment]
That seems to have fixed it.

**Comment by StormFX** [comment]
Alright. I'll include it in the next release.

**Comment by Landelor** [comment]
Thanks for your help.

**Comment by StormFX** [comment]
Should be fixed.
